.github/workflows/publish_lib.yml

```
@@ -50,3 +50,11 @@ jobs:
           git push
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create and push tag
+        run: |
+          version=$(npm pkg get version | tr -d '\n')
```

Here we get a quoted version.
```suggestion
          version=$(jq --raw-output '."version"' <package.json)
```

---

```
node --eval='console.log(require("./package.json").version);'
```
is ugly